README.md

```diff
@@ -136,30 +94,25 @@ class EmployeeDAO(BaseDAO[Employee]):
 Once `Models` and `Data Access Objects` (DAOs) are provided, you can use `Transactions` to operate the `Models`:
 
 ```python
-
-async def alter_employees():
-    # instantiate the Transaction and register the DAOs EmployeeDAO
-    # to deal with Employee models
-    t = Transaction()
-    t.register_dao(EmployeeDAO(Employee))
-
-    # retrieve John Doe's Employee model, that has a known primary key 1
-    john = await t.get(Employee, 1)   # Employee(key=1, name="John Doe", age=30, address="The Netherlands")
-    john.address = "Brazil"
-
-    # create a new Employee model Jay Pritchett in local memory
-    jay = Employee(name="Jay Pritchett", age=65, address="California")
-    # tell the transaction to add the new employee to its context
-    t.add(jay)
-
-    # persist all changes on the remote server
-    await t.commit()
-
-await alter_employees()
-
+# instantiate the Transaction and register the DAOs EmployeeDAO
+# to deal with Employee models
+transaction = Transaction()
+transaction.register_dao(EmployeeDAO(Employee))
+
+# retrieve John Doe's Employee model, that has a known primary key 1
+john = await transaction.get(Employee, 1)
+print(john) # Employee(key=1, name="John Doe", age=30, address="The Netherlands")
+
+# create a new Employee model Jay Pritchett in local memory
+jay = Employee(name="Jay Pritchett", age=65, address="California")
+# tell the transaction to add the new employee to its context
+transaction.add(jay)
```

Comment:
Se der pra fazer um exemplo com um modelo relacionado seria massa. Tem muita feature bacana que está escondida desse README.
```suggestion
# create new Employees in local memory
jay = Employee(name="Jay Pritchett", age=65, address="California")
manny = Employee(name="Manuel Delgado", age=22, address="Florida")
# tell the transaction to add the new employees to its context
transaction.add(jay)
transaction.add(manny)
```

Reply:
Thanks, I will add the suggestion.

I was thinking of an example in which I would have relational data on top of the same model (to keep the number of different models minimal in the example). Then I realized that this is currently a limitation, as you can't do anything like this
```python
class Employee(BaseModel):
    ...
    colleagues: FrozenSetModelField(Employee)
```
because the class `Employee` is not yet defined.

There is a dirty hack, namely the following
```python
class Employee(BaseModel):
    ...
    colleagues: FrozenSetModelField(BaseModel)
```
but I would like to avoid placing a hack right in the `README`. I much prefer implementing a proper feature for it, something that would support, for example, passing the model name as a string:
```python
class Employee(BaseModel):
    ...
    colleagues: FrozenSetModelField("Employee")
```
which is the approach other ORMs use for such use case.

Then I will keep this out of this PR for now and add a new feature in a separate one.